includes/class-newspack-popups.php

```diff
@@ -844,10 +844,12 @@ public static function api_config_missing_notice() {
 	}
 
 	/**
-	 * Is the user an admin user?
+	 * Is the user an admin/editor/author/contributor user?
+	 * If so, prompts will be shown to these users while logged in, but analytics
+	 * will not be fired for them.
 	 */
 	public static function is_user_admin() {
-		return is_user_logged_in() && current_user_can( 'edit_others_pages' );
+		return is_user_logged_in() && current_user_can( 'edit_posts' );
```

Comment:
I wonder if we should skip analytics for anyone other than admin and editor. Listings customers, for example, have `edit_posts` capability.

Reply:
I had the same concern. In the case of listings customers, it's a bit more complicated—they actually only have `edit_posts` when it comes to accessing the block editor, otherwise we block that capability for all other purposes (so that they don't have access to the WP admin dashboard or other parts of the site that `edit_posts` would normally grant). But I am concerned that custom user types on other sites may have `edit_posts` but should be tracked as readers.

What if we were to add a filter to allow other plugins to alter the capability we check to determine which users shouldn't be tracked?

Reply:
I think that's the way to go. For most sites `edit_others_pages` should be the right capability to check, but sites can choose to block analytics for a wider set of capabilities if needed. 2f8f3fe

Reply:
Thank you for clarifying the listings customer mechanism!

> For most sites edit_others_pages should be the right capability to check, but sites can choose to block analytics for a wider set of capabilities if needed.

I agree that's the best solution!